site/content/extend/plugins/ha.md

```diff
@@ -0,0 +1,61 @@
+---
+title: High Availabilty
+date: 2018-07-10T00:00:00-05:00
+subsection: Plugins (Beta)
+weight: 60
+---
+
+# High Availability
+
+Mattermost servers have the ability to run in High Availability (HA) mode, meaning a cluster of Mattermost app servers running together as a single Mattermost deployment. This document covers best practices for writing plugins so that they work in HA and how to deploy plugins to HA setups.
+
+## Writing Plugins
+
+It is imporant that all plugins consider HA environments when being built.
```

Comment:
It's important and a best practice that should be followed as we want all plugins to work on all Mattermost envrionments, HA or not.

Reply:
If it should be followed by all plugins (which makes sense), then should it just be a best practice that lives in the server plugins section?

Perhaps with a mention that the best practice is because of high availability

Reply:
With the deployment section being moved to the admin docs it makes more sense under the server section